SlicerLite/SlicerLiteLib/SlicerLiteModuleWidget.py

```diff
@@ -0,0 +1,216 @@
+        SlicerUtils.showVolumeAsForegroundInSlices(volumeItem.volumeNode.GetID() if volumeItem else None)
+        SlicerUtils.showVolumeAsBackgroundInSlices(volumeItem.volumeNode.GetID() if volumeItem else None)
+        # self.itemTableView.setCurrentIndex(self.itemTableModel.indexFromItem(item))
+        currentVolumeItemId = self.itemTableModel.getVolumeIdFromVolumeItem(volumeItem)
+        self.changeSelectedRow(currentVolumeItemId)
+        # Turn 3D visibility of volume to TRUE
+        self.itemTableModel.toggleVolumeVisibility(currentVolumeItemId)
+        if volumeItem:
+            self.rotateSliceViewsToSegmentation()
+
+    def changeSelectedRow(self, selectedRowId):
+        """
+        Manually change the selected row to the selectedRowId
+        """
+        if 0 <= selectedRowId < self.itemTableModel.rowCount:
+            self.itemTableView.clearSelection()
+            self.itemTableView.selectionModel().setCurrentIndex(self.itemTableModel.index(selectedRowId, 0), qt.QItemSelectionModel.Select)
+
+    def onDeleteVolumeItem(self):
+        """
+        Called after an item is deleted
+        Set the current item to the first one
+        """
+        if self.itemTableModel.rowCount() > 0:
+            self.setCurrentVolumeItem(self.itemTableModel.getVolumeItemFromId(0))
+            self.setCurrentSelectedLineOnTableView(0)
+            self.lastSelectedRowIndex = 0
+
+    def setCurrentSelectedLineOnTableView(self, rowID):
+        """
+        Update the display columns item on the input rowID (open and close persistent editors = buttons)
+        """
+        if self.lastSelectedRowIndex >= 0:
+            self.itemTableView.closePersistentEditor(self.itemTableModel.index(self.lastSelectedRowIndex, 1))
+            self.itemTableView.closePersistentEditor(self.itemTableModel.index(self.lastSelectedRowIndex, 2))
+        self.itemTableView.openPersistentEditor(self.itemTableModel.index(rowID, 1))
+        self.itemTableView.openPersistentEditor(self.itemTableModel.index(rowID, 2))
+        self.lastSelectedRowIndex = rowID
+
+    def onTableViewItemClicked(self, modelIndex: qt.QModelIndex):
+        """
+        Update views and current item according to the clicked item
+        """
+        if not modelIndex.model():
+            self.itemTableView.clearSelection()
+            self.setCurrentVolumeItem(None)
+            return
+
+        volumeItem = modelIndex.model().item(modelIndex.row()).data(Model.VolumeItemModel.ItemUserRole)
+        # We only want to select line if user click on volume's name
+        if modelIndex.column() != 0 or modelIndex.row() == self.lastSelectedRowIndex:
+            return
+
+        # Save current shift rendering value
+        if self.lastSelectedRowIndex >= 0:
+            item = self.itemTableModel.getVolumeItemFromId(self.lastSelectedRowIndex)
+            item.shiftRenderingValue = self.shiftSliderWidget.value
```

Comment:
Wondering if you shouldn't save the min / max values as well when changing the volume

Reply:
Shift rendering is a big subject for me. I have to really go deeper to understand how the default shift slider works.